packages/sys-process/glances/glances-3.1.5.exheres-0

```diff
@@ -7,5 +7,3 @@ require glances
 SLOT="0"
 PLATFORMS="~amd64"
 
-BUGS_TO="will@worrbase.com"
```

Comment:
Don't know the whole story, but BUGS_TO was removed from recommended metadata some time ago in https://gitlab.exherbo.org/exherbo-misc/docs/-/commit/8886e87da9dfd3818892e334f2c96d83d1d6fb7d